ui/pad-coms.ts

```diff
@@ -75,6 +75,9 @@ export async function requestConfig(dev: HIDDevice) {
 
 export async function requestTestData(dev: HIDDevice) {
   const response = await getSensorTestData(dev);
+  if (response[0] !== API_COMMAND.GET_SENSOR_TEST_DATA) {
+    return null;
+  }
```

Comment:
This just kind of speaks to needing to move to a system where we handle a packet based on its first byte rather than expecting the packet we get after requesting one to be the correct response.

Reply:
Yeah, definitely something to handle soon. I think this will actually be easy to do inside the sdk where we can just not resolve the promise right away and keep watching for more packets until we see the response that we're looking for.